src/setup/defaultNodes.js

```diff
@@ -43,11 +44,12 @@ const nodeTypes = [
   pool,
   poolLane,
 ];
-const timerEventNodes = [
+const timerAndBoundaryEventNodes = [
   [startTimerEvent, 'bpmn:StartEvent', 'bpmn:TimerEventDefinition'],
   [intermediateTimerEvent, 'bpmn:IntermediateCatchEvent', 'bpmn:TimerEventDefinition'],
   [intermediateMessageCatchEvent, 'bpmn:IntermediateCatchEvent', 'bpmn:MessageEventDefinition'],
   [boundaryTimerEvent, 'bpmn:BoundaryEvent', 'bpmn:TimerEventDefinition'],
+  [boundaryMessageEvent, 'bpmn:BoundaryEvent'],
```

Comment:
@sanjacornelius You're missing just one more parameter here:
```suggestion
  [boundaryMessageEvent, 'bpmn:BoundaryEvent', 'bpmn:MessageEventDefinition'],
```

Reply:
Oops! Thanks!